drivers/bridge/setup_ip_tables.go

```diff
@@ -149,7 +149,7 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 			iptables.Raw(append([]string{"-D", chain}, dropArgs...)...)
 
 			if !iptables.Exists(table, chain, acceptArgs...) {
-				if output, err := iptables.Raw(append([]string{"-A", chain}, acceptArgs...)...); err != nil {
```

Comment:
@aboch see my explanation below and the link to the PR that made this change. So an append here is indeed needed for linked containers to work when `--icc=false`